src/module_3/module3_statistical_learning.md

```diff
@@ -0,0 +1,1368 @@
+    </tr>
+    <tr>
+      <th>2880545</th>
+      <td>33826439594116</td>
+      <td>healthcarevitamins</td>
+      <td>3643274788996</td>
+      <td>3883757174916</td>
+      <td>2021-03-03 13:57:35</td>
+      <td>2021-03-03</td>
+      <td>4</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.417186</td>
+      <td>0.114360</td>
+      <td>colief</td>
+      <td>0.000000</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>34.0</td>
+      <td>27.693045</td>
+      <td>30.0</td>
+      <td>34.0</td>
+      <td>27.451392</td>
+    </tr>
+    <tr>
+      <th>2880546</th>
+      <td>33826439594116</td>
+      <td>healthcarevitamins</td>
+      <td>3643283734660</td>
+      <td>3874925314180</td>
+      <td>2021-03-03 14:14:24</td>
+      <td>2021-03-03</td>
+      <td>7</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.417186</td>
+      <td>0.114360</td>
+      <td>colief</td>
+      <td>0.000000</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>34.0</td>
+      <td>27.693045</td>
+      <td>30.0</td>
+      <td>34.0</td>
+      <td>27.451392</td>
+    </tr>
+    <tr>
+      <th>2880547</th>
+      <td>33826439594116</td>
+      <td>healthcarevitamins</td>
+      <td>3643294515332</td>
+      <td>3906490826884</td>
+      <td>2021-03-03 14:30:30</td>
+      <td>2021-03-03</td>
+      <td>2</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.417186</td>
+      <td>0.114360</td>
+      <td>colief</td>
+      <td>0.000000</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>34.0</td>
+      <td>27.693045</td>
+      <td>30.0</td>
+      <td>34.0</td>
+      <td>27.451392</td>
+    </tr>
+    <tr>
+      <th>2880548</th>
+      <td>33826439594116</td>
+      <td>healthcarevitamins</td>
+      <td>3643301986436</td>
+      <td>3914253959300</td>
+      <td>2021-03-03 14:42:05</td>
+      <td>2021-03-03</td>
+      <td>3</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.417186</td>
+      <td>0.114360</td>
+      <td>colief</td>
+      <td>0.000000</td>
+      <td>2.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>0.0</td>
+      <td>2.0</td>
+      <td>33.0</td>
+      <td>34.0</td>
+      <td>27.693045</td>
+      <td>30.0</td>
+      <td>34.0</td>
+      <td>27.451392</td>
+    </tr>
+  </tbody>
+</table>
+<p>2880549 rows × 27 columns</p>
+</div>
+
+
+
+
+```python
+items_ordered = df_base[df_base['outcome'] == 1].groupby('order_id')['variant_id'].count().reset_index().rename(columns={"variant_id": "order_size"})
```

Comment:
the name is not great, what about: "order_size_df"

Reply:
also you can do: `df_base[df_base['outcome'] == 1].groupby('order_id').size().reset_index()` instead